sos/plugins/openvswitch.py

```diff
@@ -42,6 +43,21 @@ def setup(self):
             "/var/run/openvswitch/ovsdb-server.pid",
             "/var/run/openvswitch/ovs-vswitchd.pid"
         ])
+        # Collect ovs-vswitchd's pmd thread details in procfs.
+        pid = open("/var/run/openvswitch/ovs-vswitchd.pid").read()\
+            .splitlines()[0]
```

Comment:
Worth having in try-catch block in case the file cant be opened or read - see e.g. https://github.com/sosreport/sos/blob/master/sos/plugins/pulp.py#L38-L50 for example.